revsolver/base/openshift-templates/buildConfig-template.yaml

```diff
@@ -0,0 +1,102 @@
+apiVersion: v1
```

Comment:
@fridex , we started using object directly as argo would apply them directly , having these as template would just have template applied in cluster.

Reply:
Thanks makes sense. How are build configs treated? I cannot see any in this repo.

Reply:
actually we dont use buildconfig as builds happens through CI, but keeping them here is no harm if we ever need them.

Reply:
thanks :+1: